includes/RequestWiki/RequestWikiRequestViewer.php

```diff
@@ -240,6 +247,31 @@ public function getFormDescriptor(
 				$context->getOutput()->addHTML( Html::errorBox( $wmError ) );
 			}
 
+			if ( $permissionManager->userHasRight( $userR, 'createwiki' ) ) {
+				if ( $permissionManager->userHasRight( $userR, 'createwiki-deleterequest' ) ) {
+					$viewerCwDeleteLevel = 1;
+				} else {
+					$viewerCwDeleteLevel = ( $permissionManager->userHasRight( $userR, 'createwiki-suppressrequest' )
+						? 2 : 0 );
+				}
+				$requesterCount = $wm->getUserRequestCount( $request->requester, $viewerCwDeleteLevel, '*' );
```

Comment:
`$viewerCwDeleteLevel` should be removed, and you'll need to pass in `$userR` to do the permissions checks in getUserRequestCount. Or maybe use a flag or array to store what permissions the user has so that it is only looked up once.